subcommands/devices/list.go

```diff
@@ -180,7 +180,7 @@ func showDeviceList(dl *client.DeviceList, showColumns []string) {
 		found := false
 		for i := 0; i < len(os.Args); i++ {
 			arg := os.Args[i]
-			if (len(arg) > 2 && arg[:2] == "-p") ||  (len(arg) > 7 && arg[:7] == "--page=") {
+			if (len(arg) > 2 && arg[:2] == "-p") || (len(arg) > 7 && arg[:7] == "--page=") {
```

Comment:
just push this comit to main. no need for a review on it.

Reply:
done